protocol/query/method/getMessages.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/query/method/getMessages.json",
+  "description": "Get missing messages",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "method": {
+      "description": "[String] operation to be performed by the query",
+      "const": "getMessages"

```

Comment:
From my understanding, during yesterday discussions it was said that maybe `getMessagesByIds` (or something similar sounding) was preferable over `getMessages` as the latter is maybe too general

Reply:
Is it just the name that's supposed to change ?